Started restyling of inplace-controls

oliverguenther opened the pull request:
This commit restyles the inplace control buttons (save, send, cancel)
according to the visual.
### TODOS
- [x] Fix width of new controls
- [x] Check that this restyle fixes the offset in other browsers

![bildschirmfoto 2015-09-16 um 15 18 45](https://cloud.githubusercontent.com/assets/459462/9910571/f4c14c3a-5c9d-11e5-8789-3ffc5e3a6ab7.png)

Relevant visual: http://clients.zzmedia.net/openproject/inplace-edit/
Work package: https://community.openproject.org/projects/openproject/work_packages/details/21528/overview

oliverguenther committed:
Restyling inplace-controls

This commit restyles the inplace control buttons (save, send, cancel)
according to the visual.

TeatroIO commented:
I've prepared a stage to preview changes. <a href='http://oliverguenther-feature-inplace-buttons-restyle.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://oliverguenther-feature-inplace-buttons-restyle.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

oliverguenther commented:
Hey @HDinger , can you look into the width of the controls? I can't seem to make them consistent.

HDinger commented:
https://community.openproject.org/projects/openproject/work_packages/details/21528/overview

HDinger commented:
Hey @oliverguenther , it took me a while, but I think I figured out, why the width weren't the same. It had two reasons, I think: First the DOM was slightly different for the second button and thus styled differently. Second in IE and FF we have to take care that the fields won't get too big when the border appears on hover. I addressed these problems in https://github.com/oliverguenther/openproject/pull/1

oliverguenther commented:
@HDinger Thanks a lot! Looking into it later today.

HDinger commented:
https://community.openproject.org/work_packages/21528/activity

basiszwo commented:
@HDinger I had a look at this in chrome. looks good. Is this considered to be merged? Did you review it in all relevant browsers?

oliverguenther commented:
It's not quite ready yet:
1. Icons move on hover in  FF, however surprisingly are fine in IE 10+
   ![bildschirmfoto 2015-09-18 um 07 30 21](https://cloud.githubusercontent.com/assets/459462/9952531/2ab3cf72-5dd7-11e5-9b49-764e9fe06da1.png)
2. The visual suggests these icons (rather their hover border) should be square.

oliverguenther commented:
I've addressed both these issues and rebased one last time dev. The controls are now marginally larger and I think they now match the visual quite closely.

Checked in Chrome, FF and IE: Its mergeable from my side now.

basiszwo commented:
@mk84 can you have a look please?

## 

> Am 18.09.2015 um 07:56 schrieb Oliver Günther notifications@github.com:
> 
> I've addressed both these issues and rebased one last time dev. The controls are now marginally larger and I think they now match the visual quite closely.
> 
> Checked in Chrome, FF and IE: Its mergeable from my side now.
> 
> —
> Reply to this email directly or view it on GitHub.

mk84 commented on app/assets/stylesheets/content/_in_place_editing.sass:
```diff
@@ -235,24 +213,45 @@ a.inplace-editing--trigger-link,
+.inplace-edit--control
+  width: 1.75rem
+  height: 1.75rem
+  line-height: 1
```
I think the icons are not in the vertical middle...
When you use the same height value for the line-height (1.75rem) and remove padding bottom and top from the inside a tag, the icon will be vertically centered.

HDinger commented:
A addressed the feedback of @mk84 in https://github.com/oliverguenther/openproject/pull/2. I tested it in Chrome, FF, Safari and IE10, 11. From my side it is mergeable then.

oliverguenther commented:
I've asserted the functionality in Chrome, FF and IE10+, looks good + have rebased + will merge after travis.

oliverguenther commented:
Merging despite red travis on member specs, which are the same capybara-select2 based issues that @machisuji is currently working on.
